CHANGELOG.rst

```diff
@@ -4,6 +4,12 @@ community.aws Release Notes
 
 .. contents:: Topics
 
+v2.0.0
```

Comment:
I think the CHANGELOG.rst is somewhat autogenerated. You need to revert this and add a changelogs/fragments file